Command/ListCommand.php

```diff
@@ -75,10 +76,18 @@ protected function configure(): void
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $table = new Table($output);
-        $table->setHeaders(['Cron job', 'Description']);
+        $table->setHeaders(['Cron job', 'Description','Next Run']);
         foreach ($this->cronJobManager->getCronJobs() as $cronJob) {
-            $table->addRow([get_class($cronJob), $cronJob->getDescription()]);
+
+            if (CronExpression::isValidExpression($cronJob->getExpression())) {
+                $cronExpression = CronExpression::factory($cronJob->getExpression());
+                $nextRunDate = $cronExpression->getNextRunDate()->format('Y-m-d H:i:s');
+            }
+
+            $table->addRow([get_class($cronJob), $cronJob->getDescription(), $nextRunDate]);
         }
         $table->render();
+
+        return 0;
```

Comment:
@kusiboss use `Command::SUCCESS` instead of `0`

Reply:
There is no `Command::SUCCESS` in this version of `Symfony\Component\Console\Command`

Reply:
https://github.com/symfony/console/blob/6.1/Command/Command.php#L37